<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherWidget</class>
 <widget class="QWidget" name="OtherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>796</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>相关链接</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>RTduino仓库</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="rtduino_git_url">
           <property name="text">
            <string>https://github.com/RTduino/RTduino</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="rtduino_gitee_url">
           <property name="text">
            <string>https://gitee.com/rtduino/RTduino</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>软件帮助文档</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="doc_git_url">
           <property name="text">
            <string>https://rtduino.github.io/RTduino/#/zh/manual/adapt/bsp/stm32/stm32</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="doc_gitee_url">
           <property name="text">
            <string>https://rtduino.gitee.io/#/zh/manual/adapt/bsp/stm32/stm32</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>作者主页</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="autor_meco_url">
           <property name="text">
            <string>https://github.com/mysterywolf</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="autor_liyony_url">
           <property name="text">
            <string>https://github.com/liYony</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>快捷键</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="HotkeyLineEdit" name="lineEdit_delete"/>
      </item>
      <item row="1" column="1">
       <widget class="HotkeyLineEdit" name="lineEdit_insert"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_change">
        <property name="text">
         <string>修改引脚</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_insert">
        <property name="text">
         <string>插入引脚</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_add">
        <property name="text">
         <string>添加引脚</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="HotkeyLineEdit" name="lineEdit_add"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_delete">
        <property name="text">
         <string>删除引脚</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="HotkeyLineEdit" name="lineEdit_change"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>969</width>
       <height>363</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>系统配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_update">
        <property name="text">
         <string>启动自动检测更新</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>530</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HotkeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>otherwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
